### ISSUE TITLE
Can't compile on mac

### ISSUE THREAD

**Issue by jendas1** [Snorenotify/Snorenotify#14]
Hello,
I wanted to at least compile it, but it gives me this error:
```
/Users/jendas/Downloads/Snorenotify/src/plugins/backends/osxnotificationcenter/osxnotificationcenter.h:33:18: warning: 'OSXNotificationCenter::initialize' hides overloaded virtual
      function [-Woverloaded-virtual]
    virtual bool initialize(Snore::SnoreCore *snore);
                 ^
/Users/jendas/Downloads/Snorenotify/src/libsnore/plugins/snorebackend.h:36:18: note: hidden overloaded virtual function 'Snore::SnoreBackend::initialize' declared here: different
      number of parameters (0 vs 1)
    virtual bool initialize() override;
                 ^
/Users/jendas/Downloads/Snorenotify/src/plugins/backends/osxnotificationcenter/osxnotificationcenter.mm:34:37: error: too many arguments to function call, expected 0, have 1
    return SnoreBackend::initialize(snore);
           ~~~~~~~~~~~~~~~~~~~~~~~~ ^~~~~
/Users/jendas/Downloads/Snorenotify/src/libsnore/plugins/snorebackend.h:36:5: note: 'initialize' declared here
    virtual bool initialize() override;
    ^
```
The problem is that SnoreBackend has method initialize without parameters.
I removed the snore (as a parameter in source code) and it compiled, but the notifications in OS X notification center are not working.

**Comment by TheOneRing** [comment]
Hi thanks for the report. As I don't own a mac I can only try to keep the plugin working.
after https://github.com/Snorenotify/Snorenotify/commit/d1b54f7028b76c4ca3f869fdda3e8d6245c61bcc it should hopefully work now.

**Comment by TheOneRing** [comment]
fixed in 4ec563a5dea3f167e88d371be7fefc3b13917e66
